### PR TITLE
fix: emails not being sent to proper recipients

### PR DESCRIPTION
**Pull request by fmvilas**
This PR is fixing a bug that was sending the TSC notification email to all the people in our Mailchimp list, instead of just the ones subscribed to this kind of notification.

### CODE REVIEW

**Review by derberg — APPROVED** [review]
Thanks mate 🙏🏼

File: .github/workflows/scripts/mailchimp/index.js [hunk 1]
```diff
@@ -27,8 +27,14 @@ module.exports = async (link, title) => {
             type: 'regular',
             recipients: {
                 list_id: '6e3e437abe',
-                segments_opts: {
-                    saved_segment_id: 'tsc-voting-email'
+                segment_opts: {
```
Comment (derberg) [inline .github/workflows/scripts/mailchimp/index.js:30]:
```suggestion
                segments_opts: {
```

are you sure it is correct? 😆

Comment (fmvilas) [reply .github/workflows/scripts/mailchimp/index.js:30]: Are you? 😂

**Pull request merged**